operators/src/source/ogr_source/mod.rs

```diff
@@ -350,10 +350,13 @@ pub struct InitializedOgrSource {
 impl VectorOperator for OgrSource {
     async fn _initialize(
         self: Box<Self>,
+        path: WorkflowOperatorPath,
         context: &dyn crate::engine::ExecutionContext,
     ) -> Result<Box<dyn crate::engine::InitializedVectorOperator>> {
         // TODO: check rename of fields are valid
 
+        debug!("Initializing OgrSource with prefix: {:?}", path);
```

Comment:
prefix -> path?

Reply:
also: maybe we could add the path to the span of the operator's `CreateSpan`? Then, all logs would automatically have the path added.

Reply:
https://github.com/geo-engine/geoengine/pull/780/commits/eb4d17538a5d7b7395669e3d204ef04cf988e1d4

i tried that. CreateSpan can not take params or capture variables.